hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/types/HazelcastTypeUtils.java

```diff
@@ -86,25 +82,24 @@ public final class HazelcastTypeUtils {
         CALCITE_TO_HZ.put(SqlTypeName.TIMESTAMP_WITH_LOCAL_TIME_ZONE, QueryDataType.TIMESTAMP_WITH_TZ_OFFSET_DATE_TIME);
 
         HZ_TO_CALCITE.put(QueryDataTypeFamily.OBJECT, SqlTypeName.ANY);
+        HZ_TO_CALCITE.put(QueryDataTypeFamily.MAP, SqlTypeName.ANY);
```

Comment:
Why do we have mapping `map -> any` here, but below we're doing opposite `map -> map` conversion?

Reply:
Ok, I remember that there were some issue with IMap or Kafka, that it automatically determined wrong Map type... Don't remember any more details. 
Got you.